firefox-ios/Client/Frontend/Browser/Tabs/Models/TabModel.swift

```diff
@@ -0,0 +1,33 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+struct TabModel: Equatable {
```

Comment:
I don't know why this file appears as if it's added? It's in main as well, but shouldn't it be just changing location as the other files? There's a couple of other files like this (TabsState.swift, TabsDisplayPanel. TabTrayViewController...)

Reply:
you are right, they should not appear as new files. I have seen the same with other files and was able to fix it. I will do the same with these

Reply:
Fixed, hopefully it is not added after fixing conflicts again 😅